packages/annotations/src/add.ts

```diff
@@ -3,11 +3,11 @@
 
 import {
   addFeatures,
-  IAddFeaturesRequestOptions,
-  IAddFeaturesResult
-} from "@esri/arcgis-rest-feature-service";
+  IAddFeaturesOptions
+} from "@esri/arcgis-rest-feature-layer";
 
-import { IFeature } from "@esri/arcgis-rest-common-types";
+import { IFeature } from "@esri/arcgis-rest-types";
+import { IEditFeatureResult } from "@esri/arcgis-rest-feature-layer/dist/esm/helpers";
```

Comment:
is this not exported from `"@esri/arcgis-rest-feature-layer"`?

Reply:
it will be in rest-js `v2.0.1`